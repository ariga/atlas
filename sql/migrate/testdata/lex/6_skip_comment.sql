-- comment 1
CREATE TABLE t1(id int);
# CREATE TABLE t1(id int);

-- comment 2
# CREATE TABLE t2(id int);
CREATE TABLE t2(id int);
-- comment 3
CREATE TABLE t3(id int);

# CREATE TABLE t3(id int);

/* comment 4 */
CREATE TABLE t4(
    id int
    /* comment */
    -- comment
) ENGINE=InnoDB;

/* comment 5
*/
CREATE TABLE t5(
    id int
    /* comment */
    -- comment
) ENGINE=InnoDB;
