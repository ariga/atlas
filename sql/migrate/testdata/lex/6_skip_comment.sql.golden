CREATE TABLE t1(id int);
-- end --
CREATE TABLE t2(id int);
-- end --
CREATE TABLE t3(id int);
-- end --
CREATE TABLE t4(
    id int
    /* comment */
    -- comment
) ENGINE=InnoDB;
-- end --
CREATE TABLE t5(
    id int
    /* comment */
    -- comment
) ENGINE=InnoDB;